fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/ShowTableCreationCommand.java

```diff
@@ -0,0 +1,110 @@
+import org.apache.doris.common.ErrorReport;
+import org.apache.doris.common.util.ListComparator;
+import org.apache.doris.common.util.OrderByPair;
+import org.apache.doris.nereids.trees.plans.PlanType;
+import org.apache.doris.nereids.trees.plans.visitor.PlanVisitor;
+import org.apache.doris.qe.ConnectContext;
+import org.apache.doris.qe.ShowResultSet;
+import org.apache.doris.qe.ShowResultSetMetaData;
+import org.apache.doris.qe.StmtExecutor;
+
+import com.google.common.base.Strings;
+import com.google.common.collect.Lists;
+
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+/**
+ * show table id command
+ */
+public class ShowTableCreationCommand extends ShowCommand {
+    private static final ShowResultSetMetaData META_DATA =
+            ShowResultSetMetaData.builder()
+                .addColumn(new Column("Database", ScalarType.createVarchar(20)))
+                .addColumn(new Column("Table", ScalarType.createVarchar(20)))
+                .addColumn(new Column("Status", ScalarType.createVarchar(10)))
+                .addColumn(new Column("Create Time", ScalarType.createVarchar(20)))
+                .addColumn(new Column("Error Msg", ScalarType.createVarchar(100)))
+                .build();
+
+    private String dbName;
+    private String wild;
+
+    /**
+     * constructor
+     */
+    public ShowTableCreationCommand(String dbName, String wild) {
+        super(PlanType.SHOW_TABLE_CREATION_COMMAND);
+        this.dbName = dbName;
+        this.wild = wild;
+    }
+
+    /**
+     * get meta for show tableId
+     */
+    public ShowResultSetMetaData getMetaData() {
+        return META_DATA;
+    }
+
+    @Override
+    public ShowResultSet doRun(ConnectContext ctx, StmtExecutor executor) throws Exception {
+        if (Strings.isNullOrEmpty(dbName)) {
+            dbName = ConnectContext.get().getDatabase();
+            if (Strings.isNullOrEmpty(dbName)) {
+                ErrorReport.reportAnalysisException(ErrorCode.ERR_NO_DB_ERROR);
+            }
+        }
+
+        /* Need to implement fetching records from iceberg table */
+        List<List<Comparable>> rowSet = Lists.newArrayList();
+        // sort function rows by fourth column (Create Time) asc
+        ListComparator<List<Comparable>> comparator = null;
+        OrderByPair orderByPair = new OrderByPair(3, false);
+        comparator = new ListComparator<>(orderByPair);
+        Collections.sort(rowSet, comparator);
+        List<List<String>> resultRowSet = Lists.newArrayList();
+
+        Set<String> keyNameSet = new HashSet<>();
+        for (List<Comparable> row : rowSet) {
+            List<String> resultRow = Lists.newArrayList();
+            for (Comparable column : row) {
+                resultRow.add(column.toString());
+            }
+            resultRowSet.add(resultRow);
+            keyNameSet.add(resultRow.get(0));
+        }
+
+        ShowResultSetMetaData showMetaData = getMetaData();
```

Comment:
1) can directly use META_DATA and no need for getMetaData function.
2) there are few empty lines in code, pls remove it